<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGpuMemtestClass</class>
 <widget class="QMainWindow" name="QtGpuMemtestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUDA GPU Memtest</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtgpumemtest.qrc">
    <normaloff>:/QtGpuMemtest/ncsa.ico</normaloff>:/QtGpuMemtest/ncsa.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageBasic">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCommandLinkButton" name="quickTestButton">
          <property name="text">
           <string>Quick Test</string>
          </property>
          <property name="description">
           <string>This fast diagnostic will scan your graphics cards for any obvious memory faults.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QCommandLinkButton" name="customStressTestButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>30 Minute Stress Burn</string>
            </property>
            <property name="description">
             <string>This intensive test will push your graphics cards to their limit.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="customStressDial">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>30</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>30</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelGpuTemperatures">
            <property name="text">
             <string>GPU Temperatures:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelFanSpeeds">
            <property name="text">
             <string>GPU Fan Speeds:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarOverall">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="format">
             <string> Test Progress: %p%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAdvanced">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutGpus">
          <property name="spacing">
           <number>6</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageResults">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="labelPassFail">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Passed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="buttonReturn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Return Home</string>
          </property>
          <property name="description">
           <string>Go back to the testing screen for another test.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNoDevices">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelNoDevices">
          <property name="text">
           <string>We're sorry, we couldn't find any CUDA enabled devices on your computer. Try installing CUDA-enabled drivers then restarting this program.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionShowWizardOnStartup"/>
    <addaction name="separator"/>
    <addaction name="actionClipboardResults"/>
    <addaction name="actionExportResults"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionMonitor_temperatures_2"/>
    <addaction name="actionMonitor_fan_speeds_2"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRelist"/>
   <addaction name="separator"/>
   <addaction name="actionSwitchView"/>
   <addaction name="separator"/>
   <addaction name="actionCheckAll"/>
   <addaction name="actionCheckNone"/>
   <addaction name="separator"/>
   <addaction name="actionStartChecked"/>
   <addaction name="actionStopAll"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide...</string>
   </property>
  </action>
  <action name="actionRelist">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/loop_24x24.png</normaloff>:/QtGpuMemtest/loop_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Relist</string>
   </property>
   <property name="toolTip">
    <string>Relist devices.</string>
   </property>
  </action>
  <action name="actionStartChecked">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/play_18x24.png</normaloff>:/QtGpuMemtest/play_18x24.png</iconset>
   </property>
   <property name="text">
    <string>Start Checked</string>
   </property>
   <property name="toolTip">
    <string>Start all checked GPUs.</string>
   </property>
  </action>
  <action name="actionCheckAll">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/check_24x20.png</normaloff>:/QtGpuMemtest/check_24x20.png</iconset>
   </property>
   <property name="text">
    <string>Check All</string>
   </property>
   <property name="toolTip">
    <string>Check all devices.</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStopAll">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/stop_16x16.png</normaloff>:/QtGpuMemtest/stop_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Stop All</string>
   </property>
   <property name="toolTip">
    <string>Stop all tests on all devices.</string>
   </property>
  </action>
  <action name="actionCheckNone">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/x_21x21.png</normaloff>:/QtGpuMemtest/x_21x21.png</iconset>
   </property>
   <property name="text">
    <string>Check None</string>
   </property>
   <property name="toolTip">
    <string>Uncheck all devices.</string>
   </property>
  </action>
  <action name="actionExportResults">
   <property name="text">
    <string>Export results...</string>
   </property>
  </action>
  <action name="actionClipboardResults">
   <property name="text">
    <string>Copy results to clipboard</string>
   </property>
  </action>
  <action name="actionMonitor_temperatures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitor temperatures</string>
   </property>
  </action>
  <action name="actionMonitor_fan_speeds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitor fan speeds</string>
   </property>
  </action>
  <action name="actionAlways_start_in_advanced_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always start in advanced mode</string>
   </property>
  </action>
  <action name="actionShowWizardOnStartup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show wizard on startup</string>
   </property>
   <property name="toolTip">
    <string>Show wizard on startup. If not checked, the application will default to the advanced view.</string>
   </property>
  </action>
  <action name="actionMonitor_temperatures_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitor temperatures</string>
   </property>
  </action>
  <action name="actionMonitor_fan_speeds_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitor fan speeds</string>
   </property>
  </action>
  <action name="actionSwitchView">
   <property name="icon">
    <iconset resource="qtgpumemtest.qrc">
     <normaloff>:/QtGpuMemtest/magnifying_glass_alt_24x24.png</normaloff>:/QtGpuMemtest/magnifying_glass_alt_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Advanced View</string>
   </property>
   <property name="toolTip">
    <string>Switch to advanced view.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtgpumemtest.qrc"/>
 </resources>
 <connections/>
</ui>
